tests/transaction-crud.test.ts

```
@@ -22,9 +24,39 @@ jest.mock('../src/lib/auth.middleware', () => {
+};
+
 describe('Transaction API (CRUD)', () => {
 	beforeEach(() => {
 		jest.clearAllMocks();
+		jest.restoreAllMocks();
+		prismaMock.$transaction.mockImplementation(async (callback: unknown) => {
+			if (typeof callback !== 'function') {
+				throw new Error('Expected transaction callback');
+			}
+
+			return callback(prismaMock as never);
```

ESLint

[error] Unexpected literal in error position of callback.

(n/no-callback-literal)